lib/kubernetes-deploy/kubernetes_resource/cloudsql.rb

```diff
@@ -0,0 +1,28 @@
+module KubernetesDeploy
+  class Cloudsql < KubernetesResource
+    TIMEOUT = 30.seconds
+
+    def initialize(name, namespace, file)
+      @name, @namespace, @file = name, namespace, file
+    end
+
+    def sync
+      _, st = run_kubectl("get", type, @name)
+      @found = st.success?
+      @status = true
+      log_status
+    end
+
+    def deploy_succeeded?
+      exists?
```

Comment:
This is special to this case, as the instance name is part of the required CLI flags passed to the pod spec in the Deployment.